src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -690,6 +837,28 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
         };
     }
 
+    private _generateNewHandTipMeshes(disableNearInteraction: boolean) {
+        // populate information for near hover, pick and pinch
+        const sceneToRenderTo = this._options.useUtilityLayer ? this._options.customUtilityLayerScene || UtilityLayerRenderer.DefaultUtilityLayer.utilityLayerScene : this._scene;
+        var hoverIndexMeshTip = null;
+        var pickIndexMeshTip = null;
+        if (!disableNearInteraction) {
+            hoverIndexMeshTip = SphereBuilder.CreateSphere("IndexHoverSphere", { diameter: 1}, sceneToRenderTo);
+            hoverIndexMeshTip.scaling.set(0.1, 0.1, 0.1);

```

Comment:
nit: make the scaling values for the two spheres const and pull them out, something like
const _hoverMeshRadius = 0.1;
...
hoverIndexMeshTip.scaling.set(this._hoverMeshRadius, this._hoverMeshRadius, this._hoverMeshRadius);

Then in _onXRFrame, you can scale and use the hover radius for the grabRay length, instead of hard-coding it as well. This way, if we tweak the hover/touch radius down the road, things will just work with a tweak to the const values at the top ofthe class, without needing to dig into the code to see where the values are used.